results-service/src/main/java/uk/gov/gchq/palisade/service/results/request/ResultsResponse.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.palisade.service.results.request;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import uk.gov.gchq.palisade.Generated;
+
+import java.util.Objects;
+import java.util.Optional;
+import java.util.StringJoiner;
+
+
+/**
+ * Represents the original data that has been sent from the client to Palisade Service for a request to access data.
+ * This data will be forwarded to a set of services with each contributing to the processing of this request.
+ * This version represents the output for results-service which is the response to the original request from the client.
+ * This is the last in the sequence.   It will contain the response that is to be sent back to the client based on the
+ * original request that was sent into palisade-service.
```

Comment:
The queue pointer is only sent to the results service from the results controller. It isn't sent to the client